data_discovery_ai/ml/keywordModel.py

```diff
@@ -25,37 +9,24 @@
 from tensorflow.keras.layers import Dense, Input, Dropout  # type: ignore
 from tensorflow.keras import Sequential  # type: ignore
 from tensorflow.keras.callbacks import EarlyStopping, ReduceLROnPlateau  # type: ignore
-from tensorflow.keras import backend as K  # type: ignore
+from tensorflow.keras import backend as backend  # type: ignore
 from tensorflow.keras.models import load_model  # type: ignore
 
-import logging
 from typing import Dict, Callable, Any, Tuple, Optional, List
-from configparser import ConfigParser
 import os
 from pathlib import Path
 
-os.environ["TF_USE_LEGACY_KERAS"] = "1"
+from data_discovery_ai.ml.preprocessor import KeywordPreprocessor
+import mlflow  # type: ignore
 
-logger = logging.getLogger(__name__)
-logger.setLevel(logging.INFO)
+os.environ["TF_USE_LEGACY_KERAS"] = "1"
 
 from data_discovery_ai.config.constants import KEYWORD_FOLDER
+from data_discovery_ai import logger
 
-
-def get_class_weights(Y_train: np.ndarray) -> Dict[int, float]:
-    """
-    Calculate label weights by the frequency of a label appears in all records
-    Input:
-        Y_train: numpy.ndarray. The train set of Y
-    Output:
-        label_weight_dic: Dict[int, float]. The label weights, keys are the indexs of labels and values are the weights.
-    """
-    label_frequency = np.sum(Y_train, axis=0)
-    epsilon = 1e-6
-    label_weights = np.minimum(1, 1 / (label_frequency + epsilon))
-
-    label_weight_dict = {i: label_weights[i] for i in range(len(label_weights))}
-    return label_weight_dict
+mlflow.tensorflow.autolog()
+mlflow.set_tracking_uri("http://localhost:8080")
+mlflow.set_experiment("Keyword Classification Model")
 
 
 def focal_loss(
```

Comment:
https://peps.python.org/pep-0257/

docstring should be inside the function if you need to describe that function, I need some detail to understand the focal_loss.